themes/default/content/registry/packages/aws/installation-configuration.md

```diff
@@ -208,3 +208,73 @@ Use `pulumi config set aws:<option>` or pass options to the [constructor of `new
 | `token` | Optional | Use this to set an MFA token. It can also be sourced from the `AWS_SESSION_TOKEN` environment variable. |
+
+## Centralize your configuration
+
+While you can create your AWS configurations locally, you also have the option to centralize your configurations using [Pulumi ESC (Environments, Secrets, and Configuration)](https://www.pulumi.com/docs/pulumi-cloud/esc/). Using this service will enable you to do things like run AWS CLI commands or run the `pulumi up` command with dynamically generated credentials, removing the need to configure and manage your credentials locally.
+
+To do this, you will need to complete the following steps:
+
+### Configure OIDC between Pulumi and AWS
+
+Refer to the [Configuring OpenID Connect for AWS Guide](https://www.pulumi.com/docs/pulumi-cloud/oidc/aws/) for the step-by-step process on how to do this. Note that when adding your configuration to your environment file, you can also define your AWS region in the `environmentVariables` section:
+
+```yaml
+values:
+  aws:
+    login:
+      fn::open::aws-login:
+        oidc:
+          duration: 1h
+          roleArn: <your-oidc-iam-role-arn>
+          sessionName: pulumi-environments-session
+  environmentVariables:
+    AWS_ACCESS_KEY_ID: ${aws.login.accessKeyId}
+    AWS_SECRET_ACCESS_KEY: ${aws.login.secretAccessKey}
+    AWS_SESSION_TOKEN: ${aws.login.sessionToken}
+    AWS_REGION: <YOUR_AWS_REGION> # e.g.`ap-south-1`
+```
+
+### [Optional] Move Pulumi config to your ESC environment
+
+It was mentioned earlier in this guide that you can also set your AWS region as a Pulumi configuration value in your project's stack settings file (`Pulumi.<stack-name>.yaml`). In addition to this, there may be other values that you have defined in this file, and these values can also be centralized using Pulumi ESC. To [expose these values to Pulumi IaC](https://www.pulumi.com/docs/pulumi-cloud/esc/environments/#projecting-pulumi-config), you will need to add a second-level key called `pulumiConfig` and nest any desired values underneath it as shown below:
+
+```yaml
+values:
+  aws:
+    login:
+      fn::open::aws-login:
+        oidc:
+          duration: 1h
+          roleArn: <your-oidc-iam-role-arn>
+          sessionName: pulumi-environments-session
+  environmentVariables:
+    AWS_ACCESS_KEY_ID: ${aws.login.accessKeyId}
+    AWS_SECRET_ACCESS_KEY: ${aws.login.secretAccessKey}
+    AWS_SESSION_TOKEN: ${aws.login.sessionToken}
+  pulumiConfig: # exposes Pulumi config values to the Pulumi CLI
+    aws:region: <YOUR_AWS_REGION> # e.g.`ap-south-1`
+```
+
+Note that the configuration values under `pulumiConfig` can also be referenced directly from within your Pulumi program code. This is done using the same method to reference values from your project's stack settings file. You can see examples of how to do this in the [Accessing Configuration from Code](https://www.pulumi.com/docs/concepts/config/#code) section of the Pulumi documentation.
```

Comment:
Keeping links root-relative wherever possible is best -- we aren't yet linting for this, but we probably should: 

```suggestion
Note that the configuration values under `pulumiConfig` can also be referenced directly from within your Pulumi program code. This is done using the same method to reference values from your project's stack settings file. You can see examples of how to do this in the [Accessing Configuration from Code](/docs/concepts/config/#code) section of the Pulumi documentation.
```

Reply:
I remember having some issues with the links loading in the shorter format which is why I've done it this way. For some reason I would get a 404 when testing the root relative links. I had assumed it was because the Registry and other docs are now separated? But maybe it was a fluke. I will try again.

Reply:
ahhhh fair enough, totally missed that. I did add the shortened links and will just work under the assumption that they will work as expected when published.